src/less/components.less

```diff
@@ -48,14 +48,14 @@
 @import 'components/reveals';
 @import 'components/search';
 @import 'components/selectors';
-@import 'components/stepper'; // BETA
+@import 'components/accordion/stepper'; // BETA
```

Comment:
Can we move the import for this into `src/less/components/accordion/index.less`?

Reply:
Moved from `src/less/components.less` -> `src/less/components/accordion/index.less`

```less
@import './stepper'; // BETA
```